test/e2e/self_provisioned_elasticsearch_test.go

```diff
@@ -112,6 +116,9 @@ func getJaegerSimpleProd() *v1.Jaeger {
 			Strategy: "production",
 			Storage: v1.JaegerStorageSpec{
 				Type: "elasticsearch",
+				Options: v1.NewOptions(map[string]interface{}{
```

Comment:
@jpkrohling @pavolloffay Hasn't this changed the nature of the test, from being self-provisioned?

Reply:
Yes, it has, good catch!

Reply:
+1 This should be reverted then, otherwise it's not testing self-provisioned  ES. @kevinearls ?

Reply:
@pavolloffay I'll either revert it or apply another fix by the end of the day.